pkg/tokens/creator.go

```diff
@@ -90,11 +97,21 @@ func (t *tokenCreator) create(adminProjects, memberProjects []string, reference
 		UserID:                         uuid.Nil.String(),
 		UserName:                       "@" + t.issuer,
 		Email:                          t.issuer + "@contiamo.com",
-		RealmIDs:                       memberProjects,
+		RealmIDs:                       []string{},
 		GroupIDs:                       []string{},
 		AllowedIPs:                     []string{},
-		AdminRealmIDs:                  adminProjects,
+		AdminRealmIDs:                  []string{},
 		AuthenticationMethodReferences: []string{reference},
+		AuthorizedParty:                t.issuer,
+	}
+
+	if opts.Audience != "" {
+		requestToken.Audience = opts.Audience
+	}
```

Comment:
This `if` does not really make sense, we can initialize this fields from the argument above with no difference.

Reply:
I intended that the field is an omitEmpty, but I guess even in that case it is fine. I will update

Reply:
actually, @chris-baynes can you confirm, I recall that your JWT validator did not like omitted fields,  is it possible for us to start omitting empty non-required fields like `aud`?

Reply:
@LucasRoesler removing this particular piece of code does not make a difference at all if you initialize the struct field directly above. `omitEmpty` is unrelated to my comment.

Reply:
You are right, and I have a commit ready to remove it, but i also intended to have an omitempty and realized it was missing, hence pinging Chris to verify it is ok

Reply:
For future reference, i manually verified the tokens would work with Pantheon by using https://github.com/contiamo/dev/pull/139